### ISSUE TITLE
Feature Request: Make extras and specials visible

### ISSUE THREAD

**Issue by bojtalepenye** [jarnedemeulemeester/findroid#472]
Would it be possible to show the different behind the scenes, extras, and you know specials or storyboard animations for movies and series that didn't make it in the final cut, but you decided to download them, and put them next to the said movie/series?
I would like to see them. Right now I can only view them in the Jellyfin for android app.

Anyways that is what I wanted to request. The app is amazing as always and it is so good laterally this is the only app I use for streaming even though I have Disney+ and Netflix.
Keep up the great work my friend.

**Comment by Natanel-Shitrit** [comment]
Duplicate of:
- #292
- #391

Thank you for using Findroid, please help us keep track of issues by searching for your issue before opening a new one!

**Comment by GreatestFool** [comment]
@Natanel-Shitrit, since there is another open issue that you've linked to, I think it's best if this one is closed. If only to avoid "empty" issues remaining open.

**Comment by Natanel-Shitrit** [comment]
> @Natanel-Shitrit, since there is another open issue that you've linked to, I think it's best if this one is closed. If only to avoid "empty" issues remaining open.

It's best to add comments to expand on the issue in the existing issue so people won't open new issues and there will be a need to manage the issues, it's just a waste of time.

**Comment by GreatestFool** [comment]
Yes, and there are links to the main issue for this feature in this thread that won't disappear if it's closed. There will always be users who don't search, just as there will be users who won't look through issue threads on the main page. I get where you're coming from, but there isn't really a reason to keep this one open anymore when it won't spawn any relevant discussion, since that'll just need to be done in the main issue anyway.

But you're the contributor here, not me, you do whatever you feel is best.
